### DISCUSSION TITLE
handling a VERY simple form within an application that uses RHF elswewhere

### DISCUSSION THREAD

**Discussion by tjdam** [react-hook-form/react-hook-form#3173] [Q&A] [answered]
Hi!

This is probably more React (Next.js more specifically in my case) question, but there could be some special to RHF as I'm not familiar to how it works internally...

My question is: if I have a really simple form (like one input) that wouldn't justify bringing a library in to handle, but within a project that already uses RHF in other places, does that mean it shouldn't be any performance hit if I use it RHF in it since it's already loaded in the bundle?

I feel inclined to use RHF just for consistency sake (also it's so easy :))

**Comment by bluebill1049** [comment, marked as answer]
> My question is: if I have a really simple form (like one input) that wouldn't justify bringing a library in to handle, but within a project that already uses RHF in other places, does that mean it shouldn't be any performance hit if I use it RHF in it since it's already loaded in the bundle?

No :) if your usage is simple, the hook form is only going to cost you around 5kb.
